.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout working branch
+      uses: actions/checkout@v1
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install latest versions of pip, setuptools and wheel
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+
+    - name: Cache pip
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: pip-${{ hashFiles('setup.cfg') }}
+        restore-keys: |
+          pip-
+
+    - name: Install dependencies
+      run: |
+        python -m pip install -e .[tests]
+
+    - name: Check Black
+      run: |
+        black --line-length 119 --check .
```

Comment:
I'm using the same line lenght Django defines in their Flake8 config.

Reply:
i think we should go go 80 chars per line :laughing:

Reply:
(obviously joking)